distributed/tests/test_preload.py

```diff
@@ -61,6 +62,17 @@ def dask_setup(worker):
             assert w.foo == "setup"
 
 
+@gen_test()
+async def test_preload_manager_sequence():
+    text = """
+def dask_setup(worker):
+    worker.foo = 'setup'
+"""
+    async with Scheduler(dashboard_address=":0", preload=text) as s:
+        assert len(s.preloads) == 1
+        assert isinstance(s.preloads[0], Preload)
```

Comment:
I think it would be good to add a test that checks `[x for x in s.preloads]` as that's what's called in coiled

Reply:
added an aditional assert